### PR TITLE
Add toast on deleting cohort

### PR DESCRIPTION
**Pull request by Fidela1**
# PR Description

This PR add toast on delete cohort and add loading on delete buttons

# Description of tasks that were expected to be completed

- [x] Add toast on deleting cohort
- [x] Add loading on delete buttons

# How has this been tested?

1. Login to devpulse
2. Navigate to cohort page
3. Try to delete cohort and you will get toast tell you that cohort deleted successfully

# Number of Commits

The number of commits should not exceed **2 commits**. In case they are more than that, please take your time and squash them.

# Screenshots (If appropriate)

# Please check this Checklist before you submit your PR:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My code generate no warnings
- [ ] My test coverage meet the set test coverage threshold
- [ ] There are no vulnerabilities
- [ ] There are no conflicts with the base branch

### CODE REVIEW

**Review by teerenzo — CHANGES_REQUESTED** [review]
I can't login, make sure that we are able to login

**Review by UwaweTera — APPROVED** [review]

**Review by UwaweTera — COMMENTED** [review]
Fidela try to use remote URL for metron in this PR

**Review by UwaweTera — CHANGES_REQUESTED** [review]
Fidela try to use remote URL for metron in this PR

**Review by Munezeroolivierhugue — CHANGES_REQUESTED** [review]
fidela try to give a description of how to test it because go to cohort page is challenging

**Review by teerenzo — APPROVED** [review]

**Review by ceelogre — APPROVED** [review]

**Review by UwaweTera — APPROVED** [review]

**Review by Munezeroolivierhugue — APPROVED** [review]

File: src/containers/admin-dashBoard/DeleteCohortModal.tsx [hunk 1]
```diff
@@ -27,11 +27,13 @@ export default function DeleteCohortModal({
   const { t } = useTranslation();
   const [deleteCohortMutation, { loading }] = useMutation(DeleteCohort, {
     onError(error) {
+      removeModel();
       toast.error(error.message.toString());
     },
     onCompleted() {
       refetch();
       removeModel();
+      toast.success("Cohort deleted successfully");
```
Comment (ceelogre) [inline src/containers/admin-dashBoard/DeleteCohortModal.tsx]: Message should be translated.

**Review by ceelogre — CHANGES_REQUESTED** [review]

**Pull request merged**